collect_app/src/test/java/org/odk/collect/android/utilities/TextUtilsTest.java

```diff
@@ -22,22 +22,44 @@ public void textToHtml_BouncesNullInput() {
         Assert.assertNull(observed);
     }
 
+
+
+    @Test
+    public void markDownToHtmlEscapesBacklash() {
```

Comment:
I like the idea of escaping backlash (a strong and adverse reaction), but this should probably be back**s**lash.

Reply:
oh yeah! exactly. Thank you